openedx/openedx-platform
# feat: add library copy management command

This PR introduces the "copy" management command, which copies v1 libraries into v2 libraries.

There will be a follow-up pr to add an "uncopy" step which removes a v2 library based on it's v1 id.

How to test:
0. Have the blockstore waffle switch enabled and lib-authoring running, but not the default experience. Make sure to have a collection enabled (see lib-authoring setup docs for this). 
1. Create a V1 Library, with content and permissions
2. In the studio shell, run `./manage.py cms copy_libraries_from_v1_to_v2 'collection_uuid' 'v1_lib_id'`
3. Navigate to lib-authoring MFE. See that all libraries have been copied over.
